src/containers/api/index.js

```diff
@@ -250,24 +212,40 @@ class Api extends Component {
     return (
       <div>
         {this.props.dataUsers.length > 0 && this.userList(this.props.dataUsers)}
-        {this.state.modal === true && (
-          <ModalBackground>
-            <Modal>
-              <ModalHeader>
-                <img src={timeblue} alt="blueLogo" />
-                <p>Book Appointment</p>
-              </ModalHeader>
-              <ModalBody>
-                <p>Pick date biatch</p>
-              </ModalBody>
-              <ModalFooter>
-                <button onClick={() => this.setState({ modal: false })}>
-                  Close Modal
-                </button>
-              </ModalFooter>
-            </Modal>
-          </ModalBackground>
-        )}
+        <Modal
+          visable={this.state.showModal}
+          modalHeader="Date picking modal"
+          modalFooterSubmit="Pick Date"
+          modalFooterCancel="cancel pick"
+          onSubmit={() => this.setState({ showModal: false })}
+        >
+          <div>
+            <p>here kemur calander</p>
+          </div>
+        </Modal>
+        <Modal
+          visable={this.state.showModal2}
+          modalHeader="Booking modal"
+          modalFooterSubmit="Book time"
+          modalFooterCancel="cancel booking"
+          onSubmit={() => this.setState({ showModal2: false })}
+        >
+          <div>
+            <div>
+              Start <img alt="timeblue" src={timeBlue} />
```

Comment:
alt text is used to describe the photo if the image doesn't load or if a screenreader is reading the page, so `timeblue` is not a good description.

Reply:
Maybe you can use `aria-hidden="true"`or something like that and skip the alt text, or just come up with a better description